WebHostLib/templates/userContent.html

```diff
@@ -33,10 +49,14 @@ <h2>Your Rooms</h2>
                             <tr>
                                 <td><a href="{{ url_for("view_seed", seed=room.seed.id) }}">{{ room.seed.id|suuid }}</a></td>
                                 <td><a href="{{ url_for("host_room", room=room.id) }}">{{ room.id|suuid }}</a></td>
-                                <td>{{ room.seed.slots|length }}</td>
+                                <td>
+                                    <div title="{{ games(room.seed.slots) }}">
+                                        {{ room.seed.slots|length }}
+                                    </div>
+                                </td>
```

Comment:
Minor nitpick. Is there any reason we don't just put the title on the td?
This would allow the full cell to show the title rather than just the div area (because the cell padding)

```suggestion
                                <td title="{{ games(room.seed.slots) }}">
                                    {{ room.seed.slots|length }}
                                </td>
```